Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -8,16 +8,34 @@ import UIKit
 internal final class CommentsViewController: UITableViewController {
   // MARK: - Properties
 
+  private lazy var commentComposer: CommentComposerView = {
+    let frame = CGRect(x: 0, y: 0, width: view.frame.width, height: 80)
+    let view = CommentComposerView(frame: frame)
+    return view
+  }()
+
   fileprivate let viewModel: CommentsViewModelType = CommentsViewModel()
 
   // MARK: - Lifecycle
 
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
+    tableView.keyboardDismissMode = .interactive
+    tableView.estimatedRowHeight = UITableView.automaticDimension
+    tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cellId")
```

Comment:
Will probably need to replace this after the [cell card](https://kickstarter.atlassian.net/browse/NT-1892) is merged.